### PR TITLE
31 implement modal form

### LINKED ISSUE #31: Implement modal form
**ACCEPTANCE CRITERIA**

Create the UI of the modal form that would contain the following elements 

- Input for `firstName` (string)
- input for `lastName` (string)
- input for `email` (string)
- checkbox for `isConfirmed` (boolean)
- select with options for `diaretyType` (string)
- textarea for `more` (string)

**TASKS**
- Create design with the required amount of fields
- Create a `handleSubmit` function in the //HANDLE FUNCTION section that is connected to the `onSubmit` function of the `form` tag element
- Make sure that each input/select/textarea have related label to describe them
- Create a button at the end of the form with a `type="submit"` 
- Create a state hook that contain a default value object of 
```js
{
  firstName: "",
  lastName: "",
  email: "",
  isConfirmed: false,
  diaretyType: "",
  more: ""
}
```
- Create a `handleChange` that will be connected to each `onChange` function of the input/select/etc
- Confirm that whenever we write text on each input or select, it should update the correct value of the state attributed to it.

### PR DESCRIPTION
**Pull request by Isynopoulos1**
I added the modal form with styling and responsive rules.
I implemented the handle functions.

### CODE REVIEW

File: src/components/organisms/modal/Modal.jsx [hunk 1]
```diff
@@ -1,16 +1,143 @@
...
+import SendBtn from "@atoms/sendBtn/SendBtn";
 
 const Modal = ({ onClose }) => {
+  const translate = useSelector(state => getTranslate(state.localize));
+  //HOOKS
+  const [formData, setFormData] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    attendance: "",
```
Comment (erwanriou) [inline src/components/organisms/modal/Modal.jsx]: incorrect field name

File: src/components/organisms/modal/Modal.jsx [hunk 2]
```diff
@@ -1,16 +1,143 @@
...
+    dietaryType: "Regular menu",
+    companions: "",
```
Comment (erwanriou) [inline src/components/organisms/modal/Modal.jsx]: remove this field

File: src/components/organisms/modal/Modal.jsx [hunk 3]
```diff
@@ -1,16 +1,143 @@
...
+            />
+          </label>
+          <br />
+
+          <RadioGroup>
+            <label>{translate("modal.text.6")}</label>
+            <RadioLabel>
+              <input
+                type="radio"
+                name="attendance"
```
Comment (erwanriou) [inline src/components/organisms/modal/Modal.jsx]: incorrect field name

File: src/components/organisms/modal/Modal.jsx [hunk 4]
```diff
@@ -1,16 +1,143 @@
...
+                onChange={handleChange}
+                required
+              />
+              <span></span>
+              {translate("modal.text.7")}
+            </RadioLabel>
+            <RadioLabel>
+              <input
+                type="radio"
+                name="attendance"
```
Comment (erwanriou) [inline src/components/organisms/modal/Modal.jsx]: incorrect field name

File: src/components/organisms/modal/Modal.jsx [hunk 5]
```diff
@@ -1,16 +1,143 @@
...
+          <br />
+          <label>
+            {translate("modal.text.9")}
+            <select
+              name="dietaryType"
+              value={formData.dietaryType}
+              onChange={handleChange}
+            >
+              <option value="Regular menu">{translate("modal.text.10")}</option>
+              <option value="Gluten-free">{translate("modal.text.11")}</option>
```
Comment (erwanriou) [inline src/components/organisms/modal/Modal.jsx]: please make sure the values are like this :

`GLUTEN_FREE`

**Review by erwanriou — CHANGES_REQUESTED** [review]
some few changes

File: src/components/organisms/modal/Modal.jsx (hunk 1 above)
Comment (Isynopoulos1) [reply src/components/organisms/modal/Modal.jsx]: done

File: src/components/organisms/modal/Modal.jsx (hunk 2 above)
Comment (Isynopoulos1) [reply src/components/organisms/modal/Modal.jsx]: done

File: src/components/organisms/modal/Modal.jsx (hunk 4 above)
Comment (Isynopoulos1) [reply src/components/organisms/modal/Modal.jsx]: ok

File: src/components/organisms/modal/Modal.jsx (hunk 5 above)
Comment (Isynopoulos1) [reply src/components/organisms/modal/Modal.jsx]: Should I write the 3 values in capital letters?  (GLUTEN_FREE, REGULAR MENU, VEGAN)

Comment (Isynopoulos1) [reply src/components/organisms/modal/Modal.jsx]: I fixed the errors, is ready to review it

File: src/assets/translations/es.translations.json [hunk 6]
```diff
@@ -36,9 +36,9 @@
   "modal.text.7": " Sí",
   "modal.text.8": " No",
   "modal.text.9": "Selecciona tu tipo de dieta",
-  "modal.text.10": "Menú normal",
```
Comment (erwanriou) [inline src/assets/translations/es.translations.json]: This is wrong i don't want to change this

File: src/assets/translations/en.translations.json [hunk 7]
```diff
@@ -36,9 +36,9 @@
   "modal.text.7": " Yes",
   "modal.text.8": " No",
   "modal.text.9": "Select your dietary type",
-  "modal.text.10": "Regular menu",
```
Comment (erwanriou) [inline src/assets/translations/en.translations.json]: I don't want you to change the translation string but the `values` instead you have to revert this change please

**Review by erwanriou — CHANGES_REQUESTED** [review]

File: src/components/organisms/modal/Modal.jsx (hunk 5 above)
Comment (erwanriou) [reply src/components/organisms/modal/Modal.jsx]: you did it wrong. you changed the `translations`

File: src/assets/translations/es.translations.json (hunk 6 above)
Comment (Isynopoulos1) [reply src/assets/translations/es.translations.json]: I changed the values already

File: src/assets/translations/en.translations.json (hunk 7 above)
Comment (Isynopoulos1) [reply src/assets/translations/en.translations.json]: I reverted this as well

Comment (Isynopoulos1) [reply src/assets/translations/en.translations.json]: I added the selector and I changed the button style in the form

**Review by erwanriou — APPROVED** [review]
Nice job
